src/api/status/public/js/utils/toast.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * UI theme for toast
+ * @typedef {("danger" | "success" | "info" | "warning" | "primary")} ToastVariant
+ */
+
+/**
+ * @returns {HTMLElement} the sample toast to clone from
+ */
+const querySampleToastEl = () => document.getElementById('sample-toast');
+/**
+ * @returns {HTMLElement} the container for all stacking toast elements
+ */
+const queryToastContainer = () => document.getElementById('toast-container');
+
+/**
+ * Make a new empty toast node from the sample toast node
+ *
+ * @returns {HTMLElement} an empty toast element
+ */
+const cloneDummyToastNode = () => {
+  // get the sample toast
+  const sampleToastEl = querySampleToastEl();
+
+  // clone from the sample
+  const clonnedToastEl = sampleToastEl.cloneNode(true);
+  clonnedToastEl.removeAttribute('id');
+  clonnedToastEl.removeAttribute('aria-hidden');
+
+  // return the new toast element
+  return clonnedToastEl;
+};
+
+/**
+ * Create a toast element and modify the content
+ *
+ * @param {string} message to display
+ * @param {ToastVariant} variant for toast UI theme
+ * @returns {HTMLElement} styled toast element with custom message
+ */
+const createToastNode = (message, variant) => {
```

Comment:
Is it worth defining a default value here (info?)

Reply:
Let's do that.